india_compliance/gst_india/doctype/gst_settings/gst_settings.json

```diff
@@ -298,12 +300,24 @@
    "fieldname": "archive_party_info_days",
    "fieldtype": "Int",
    "label": "Archive Party Information (In Days)"
+  },
+  {
+   "fieldname": "e_invoice_auto_retry_after",
+   "fieldtype": "Datetime",
+   "label": "e-Invoice Auto Retry After"
```

Comment:
do you really want user to be able to edit this?

Reply:
ideally, field should be hidden. it could even ideally be a "global". If it is being set in GST Settings, `update_modified` needs to be False in order to prevent unnecessary `TimeStampMismatchError`s